common/powershell/hyperv/hyperv.go

```diff
@@ -53,6 +53,22 @@ $ip
 	return cmdOut, err
 }
 
+//func CreateVirtualHardDiskDrive(vmName string, diskPath string, diskSize int64, diskBlockSize int64, generation uint) (uint, uint, error) {
```

Comment:
You probably want to delete this part. This is some scratch code from when I thought about creating VMs with a separate function to make adding additional VHDs easier. Looks like this feature was implemented in https://github.com/hashicorp/packer/pull/5941